### ISSUE TITLE
Add Subscription section

### ISSUE THREAD

**Issue by marko-baric-ca** [daytonaio/enterprise-docs#162]
Add pages about subscriptions, plans, payment options. This section needs to cover details about user subscription and payment options. Also cancelling plan and upgrading plans.

**Comment by marko-baric-ca** [comment]
![image](https://github.com/user-attachments/assets/cd37473d-4ba2-4ef1-ad5d-a69bbb0dc5f4)

**Comment by marko-baric-ca** [comment]
Explain the pricing model to user:

- ree trial with limited usage and slow machine
- paid plans with more usage
- special plan for GPU-powered containers
- add-ons for even more usage or even more AI

**Comment by ivan-burazin** [comment]
@marko-baric-ca suggestion we have info about plans how to upgrade and pay, but not details about the plans or prices themselves

**Comment by stefanicjuraj** [comment]
@ivan-burazin @marko-baric-ca 

This is one of the options without displaying the prices.

Each plan would then further be described in its own section regarding the CPU, memory, usage,... and their available add-on packages.

Let me know what you think and whether this should be changed to include the prices of each subscription plan.

<img width="779" alt="subscription" src="https://github.com/user-attachments/assets/8dc4bda8-1a3e-49c4-b9ff-2fd1f0f6a8d1">

**Comment by ivan-burazin** [comment]
@stefanicjuraj i would remove the plans themself from the docs.

**Comment by marko-baric-ca** [comment]
Yes, let's remove the plan details from the docs. We can just generally explain that plans define how moch power and usage time you get from your environments. But the exact plans and their prices can be left out.

**Comment by ivan-burazin** [comment]
@stefanicjuraj lmk when updated
